agents/PCS-Large-Swap/src/agent.spec.ts

```diff
@@ -0,0 +1,202 @@
+import { ethers, Finding, FindingSeverity, FindingType, HandleTransaction } from "forta-agent";
+import { provideBotHandler } from "./agent";
+import { encodeParameter } from "forta-agent-tools/lib/utils";
+import { TestTransactionEvent, createAddress, MockEthersProvider } from "forta-agent-tools/lib/tests";
+import { getPancakePairCreate2Address } from "./utils";
+import { ERC20ABI, PANCAKE_PAIR_ABI } from "./constants";
+import { BigNumber } from "ethers";
+import DataFetcher from "./data.fetcher";
+
+const createFinding = (
```

Comment:
this was already defined in utils, why define a copy with different syntax here?
Its mere existence says that you are not testing the original code, then what is the point of writing a test?

Reply:
Done.